workspaces/blackduck/plugins/blackduck-backend/package.json

```diff
@@ -34,21 +34,21 @@
   },
   "dependencies": {
     "@backstage-community/plugin-blackduck-common": "workspace:^",
-    "@backstage/backend-defaults": "^0.7.0",
-    "@backstage/backend-plugin-api": "^1.1.1",
+    "@backstage/backend-defaults": "^0.8.0",
```

Comment:
It would be better to revert these changes to `@backstage` packages and do a proper version bump for the entire workspace in a dedicated PR.